packages/core/src/Select/Select.tsx

```diff
@@ -72,6 +73,12 @@ export interface HvSelectProps<
   autoComplete?: string;
   /** Whether the width of the select panel can vary independently. */
   variableWidth?: boolean;
+  /**
+   * Properties passed on to the input element.
+   */
+  inputProps?: React.InputHTMLAttributes<HTMLInputElement>;
```

Comment:
I think we should abstract this away from the user. From the users' perspective, there shouldn't be an "underlying input element" (`HvSelect` it the input itself, like HTML's `<select>`).

For now, we could extend from the native element (`Pick<HTMLInputElement, "form">`), but in the future, we should hoist this and any other relevant form props to the `HvFormElement` props